### PR TITLE
fix issue #1235 - In docs, always capitalize Newsboat and Podboat

### PR DESCRIPTION
**Pull request by panvicka**
Hi, 
I hope I got them all! 

I only touched the /doc. There are some more randomly in code comments/file headers. 

nice DOC btw :+1:

### CODE REVIEW

File: doc/configcommands.dsv [hunk 1]
```diff
@@ -80,7 +80,7 @@ newsblur-passwordeval||<command>||""||Another secure alternative, is providing y
 newsblur-url||<url>||"https://newsblur.com"||Configures the URL where the NewsBlur instance resides.||newsblur-url "https://localhost"
 notify-always||[yes/no]||no||If set to `no`, notifications will only be made when there are new feeds or articles. If set to `yes`, notifications will be made regardless.||notify-always yes
 notify-beep||[yes/no]||no||If set to `yes`, then the speaker will beep on new articles.||notify-beep yes
-notify-format||<string>||"newsboat: finished reload, %f unread feeds (%n unread articles total)"||Format string that is used for formatting notifications. See the chapter on format strings for more information.||notify-format "%d new articles (%n unread articles, %f unread feeds)"
+notify-format||<string>||"Newsboat: finished reload, %f unread feeds (%n unread articles total)"||Format string that is used for formatting notifications. See the chapter on format strings for more information.||notify-format "%d new articles (%n unread articles, %f unread feeds)"
```
Comment (Minoru) [inline doc/configcommands.dsv:83]: Please revert this change: the third column here is the default value of the setting, and since this PR doesn't (and shouldn't) change the default, it shouldn't change this line too.

File: doc/manpage-newsboat.asciidoc [hunk 2]
```diff
@@ -4,7 +4,7 @@ Alexander Batischev <eual.jp@gmail.com>
 
 == NAME
 
-newsboat - an RSS/Atom feed reader for text terminals
+Newsboat - an RSS/Atom feed reader for text terminals
```
Comment (Minoru) [inline doc/manpage-newsboat.asciidoc:7]: This is a manpage, and this line describes a command. The rest of the document talks of the program. Confusing, I know, but that's how it is :) Please revert this change.

File: doc/manpage-newsboat.asciidoc [hunk 3]
```diff
@@ -150,4 +150,4 @@ include::chapter-environment-variables.asciidoc[]
 
 == SEE ALSO
 
-podboat(1)
+Podboat(1)
```
Comment (Minoru) [inline doc/manpage-newsboat.asciidoc:153]: Manpage names are command names, not "product" names, so please revert this change.

File: doc/manpage-podboat.asciidoc [hunk 4]
```diff
@@ -4,7 +4,7 @@ Alexander Batischev <eual.jp@gmail.com>
 
 == NAME
 
-podboat - a podcast download manager for text terminals
+Podboat - a podcast download manager for text terminals
```
Comment (Minoru) [inline doc/manpage-podboat.asciidoc:7]: Please revert this as well (see above about first line in manpage).

File: doc/manpage-podboat.asciidoc [hunk 5]
```diff
@@ -79,4 +79,4 @@ include::chapter-environment-variables.asciidoc[]
 
 == SEE ALSO
 
-newsboat(1)
+Newsboat(1)
```
Comment (Minoru) [inline doc/manpage-podboat.asciidoc:82]: Please revert this as well (see above about manpage names).

File: doc/newsboat.asciidoc [hunk 6]
```diff
@@ -1130,7 +1130,7 @@ Dialog List:<<dialogs-title-format,+dialogs-title-format+>>:%N %V - Dialogs
 [frame="all", grid="all", format="dsv", options="header", cols="30,70"]
 |======================================================================
 Identifier:Meaning
-[[common-title-format-N]]<<common-title-format-N,+N+>>:Name of the program, i.e. "newsboat"
+[[common-title-format-N]]<<common-title-format-N,+N+>>:Name of the program, i.e. "Newsboat"
```
Comment (Minoru) [inline doc/newsboat.asciidoc:1133]: We'll need to change some code to make this true, so for now, please revert this change.

File: doc/chapter-firststeps.asciidoc [hunk 7]
```diff
@@ -1,10 +1,10 @@
-After you've installed newsboat, you can run it for the first time by typing
+After you've installed Newsboat, you can run it for the first time by typing
 `newsboat` on your command prompt. This will bring you the following message:
 
 ----
 Error: no URLs configured. Please fill the file /home/ak/.newsboat/urls with RSS feed URLs or import an OPML file.
 
-newsboat 2.10
+Newsboat 2.10
```
Comment (Minoru) [inline doc/chapter-firststeps.asciidoc:7]: This is a listing of the program's output, and we'll have to change some code to change the listing. For now, please revert this change.

**Review by Minoru — CHANGES_REQUESTED** [review]
Great, thank you very much!

There are a couple places that confused you. I left a comment under each; please revert those changes.

**Comment by Minoru** [comment]
> I only touched the /doc. There are some more randomly in code comments/file headers.

Yeah, that's perfectly fine. Comments are nowhere as important, don't bother with them.

There are two places where the code has to be updated in order to update the doc. If you'd like to dive deeper and work on that, just tell me; otherwise I'm fine with merging this after you revert the changes I commented on.

**Review by Minoru — APPROVED** [review]
Excellent! :+1:

Thank you very much for working on this! I'll merge once the CI builds finish.

**Comment by Minoru** [comment]
@panvicka, if you'd like to work on Newsboat some more, #910 might be a good thing to fix regarding docs. It requires some passing familiarity with the outline of the doc, but you probably have that from working on this PR. If you'd like to sink your teeth into code instead, there's some stuff [labeled "good first issue"](https://github.com/newsboat/newsboat/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22). You can also pick any other issue; I'm happy to label your PRs "hacktoberfest-accepted" if that's the goal you're working towards.

Thanks for the pull request once again!

**Comment by panvicka** [comment]
> @panvicka, if you'd like to work on Newsboat some more, #910 might be a good thing to fix regarding docs. It requires some passing familiarity with the outline of the doc, but you probably have that from working on this PR. If you'd like to sink your teeth into code instead, there's some stuff [labeled "good first issue"](https://github.com/newsboat/newsboat/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22). You can also pick any other issue; I'm happy to label your PRs "hacktoberfest-accepted" if that's the goal you're working towards.
> 
> Thanks for the pull request once again!

Thank you. Labeling the PR with "hacktoberfest-accepted" would be great! I will have a look at the other issues...

**Comment by Minoru** [comment]
Ah, actually just merging or even approving a PR [is enough](https://hacktoberfest.digitalocean.com/details). So this one already counts, and any further ones will count too since this repo has a "Hacktoberfest" topic :ok_hand:
